va/va.go

```diff
@@ -393,6 +393,11 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 		// Intercept DialContext in order to connect to the IP address we
 		// select.
 		DialContext: dialer.DialContext,
+		// These match the values from DefaultRoundTripper in net/http.
+		MaxIdleConns:          100,
```

Comment:
Given we want to use a new connection each time, and we explicitly disable KeepAlives, don't we want this to be 0?

Reply:
And if so do we need a `IdleConnTimeout` set at all?